ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico.xslt

```diff
@@ -688,6 +694,9 @@
             <xsl:apply-templates />
          	</rico:TypeRelation>
       </rico:thingIsTargetOfTypeRelation>
+	  <rico:hasOrHadCategory>
```

Comment:
IMHO it should rather be `rico:hasOrHadCorporateBodyType` here (we have a TypeRelation that has object a CorporateBodyType; and when the legalStatus element has not date or descriptive note, it is converted into a rico:hasCorporateBodyType